configure-nonroot.sh

```diff
@@ -17,15 +17,20 @@ IRONIC_GROUP="ironic"
 
 # most containers mount /shared but dnsmasq can live without it
 mkdir -p /shared
+mkdir -p /data
+mkdir -p /conf
+
 chown "${IRONIC_USER}":"${IRONIC_GROUP}" /shared
-chown "${IRONIC_USER}":"${IRONIC_GROUP}" /data
-chown "${IRONIC_USER}":"${IRONIC_GROUP}" /conf
+# Change from IRONIC_USER to root user for ironic-proxy
```

Comment:
We should not diverge from upstream in such a way, I'd prefer you leave this file alone and really see what is wrong with ironic-proxy. Are we running it with right users/groups?

Reply:
Fixed.